CONTRIBUTING.md

```diff
@@ -1,123 +1,55 @@
 # CONTRIBUTING GUIDELINES
-
-Oh-My-Bash is a community-driven project. Contribution is welcome, encouraged and appreciated.
-It is also essential for the development of the project.
+Oh-My-Bash is Free and Open Source project under the terms of GNU General Public License v2.0 relying on contributions from third parties.
```

Comment:
@Kreyren Can you please update to a new contributing proposal to match with our current License?
Thanks.

Reply:
Now we are licensing OMB with the MIT license.